pkg/features/kube_features.go

```diff
@@ -292,6 +292,12 @@ const (
 	// Make the kubelet use shutdown configuration based on pod priority values for graceful shutdown.
 	GracefulNodeShutdownBasedOnPodPriority featuregate.Feature = "GracefulNodeShutdownBasedOnPodPriority"
 
+	// owner: @mcluseau
+	// alpha: v1.32
+	//
+	// Make the kubelet and kube-proxy use a dialer tries every IP returns by net.LookupHost in a random order.
+	HADialer featuregate.Feature = "HADialer"
```

Comment:
you need a KEP approved by the corresponding SIG

Reply:
ok, removing the feature gate

Reply:
I mean ... I think you should go through KEP to get buy in and use the feature gate to rollout the change safely ...